pkgs/development/compilers/urweb/default.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
     export PGHEADER="${postgresql}/include/libpq-fe.h";
     export MSHEADER="${libmysqlclient}/include/mysql/mysql.h";
     export SQHEADER="${sqlite.dev}/include/sqlite3.h";
+    export ICU_INCLUDES="-I${icu.dev}/include";
```

Comment:
For quick crosschecking see configure script in upstream:

https://github.com/urweb/urweb/blob/master/configure.ac#L94-L98